src/current/v24.1/user-defined-functions.md

```diff
@@ -14,8 +14,10 @@ A user-defined function (UDF) is a named function defined at the database level
 
 The basic components of a user-defined function are a name, list of arguments, return type, volatility, language, and function body.
 
-- An argument has a _mode_ and a _type_. CockroachDB supports the `IN` argument mode. The type can be a built-in type, [user-defined `ENUM`]({% link {{ page.version.version }}/enum.md %}), or implicit record type. CockroachDB **does not** support default values for arguments.
-- The return type can be a built-in [SQL type]({% link {{ page.version.version }}/data-types.md %}), user-defined [`ENUM`]({% link {{ page.version.version }}/enum.md %}), [`RECORD`]({% link {{ page.version.version }}/create-function.md %}#create-a-function-that-returns-a-record-type), PL/pgSQL [`REFCURSOR`]({% link {{ page.version.version }}/plpgsql.md %}#declare-cursor-variables) type, implicit record type, or `VOID`.
+- An argument has a _mode_ and a _type_.
+  - CockroachDB supports the `IN` (default), `OUT`, and `INOUT` argument modes. For an example, see [Create a function that uses `OUT` and `INOUT` parameters]({% link {{ page.version.version }}/create-function.md %}#create-a-function-that-uses-out-and-inout-parameters).
+  - The type can be a built-in type, [user-defined `ENUM`]({% link {{ page.version.version }}/enum.md %}) or [composite]({% link {{ page.version.version }}/create-type.md %}#create-a-composite-data-type) type, or implicit record type. A type can have a [`DEFAULT` value]({% link {{ page.version.version }}/default-value.md %}).
```

Comment:
Hm, the link for `DEFAULT` value leads to the DEFAULT value used in CREATE TABLE, and at the very top of that page it says:
> The DEFAULT value [constraint](https://deploy-preview-18495--cockroachdb-docs.netlify.app/docs/v24.1/constraints) specifies a value to write into the constrained column if one is not defined in an INSERT statement.

I think I'd drop the link here since it could be confusing - like "how does CREATE TABLE definition connect with routine arguments?".